src/coreservices.cpp

```diff
@@ -423,6 +423,27 @@ void CoreServices::initialize(QApplication* pApp) {
 
     m_pTouchShift = std::make_unique<ControlPushButton>(ConfigKey("[Controls]", "touch_shift"));
 
+    // The following UI controls must be created here so that controllers can bind to them
+    // on startup.
+    m_uiControls.clear();
+    const std::vector<ConfigKey> uiKeys = {
+            ConfigKey("[Master]", "skin_settings"),
+            ConfigKey("[Microphone]", "show_microphone"),
+            ConfigKey(VINYL_PREF_KEY, "show_vinylcontrol"),
+            ConfigKey("[PreviewDeck]", "show_previewdeck"),
+            ConfigKey("[Library]", "show_coverart"),
+            ConfigKey("[Master]", "maximize_library"),
+            ConfigKey("[Samplers]", "show_samplers"),
+            ConfigKey("[EffectRack1]", "show"),
+            ConfigKey("[Skin]", "show_4effectunits"),
+            ConfigKey("[Master]", "show_mixer"),
+    };
+
```

Comment:
```suggestion
    m_uiControls.reserve(uiKeys.size());
```

Reply:
For a function that is called once per instance, and iterates over ~10 items... this is a little silly but ok